src/provider/store/Actions.ts

```diff
@@ -50,8 +52,8 @@ export type ActionMap<T extends Action = Action> = {
 export const Actions: ActionMap = {
     [Action.CREATE]: (state: Immutable<RootState>, action: CreateNotification): RootState => {
         const {notification} = action;
-
-        notificationStorage.setItem(notification.id, JSON.stringify(notification));
+        const storage = Injector.get<'STORAGE'>(Inject.STORAGE);
```

Comment:
Makes me uneasy that we're not doing our usual constructor injection here, but can believe this is the only possible way. Able to confirm that there isn't a sensible alternative?

Reply:
I didn't like it either.  The problem is that because we are not in a class the injector gets a bit silly.  In fact, even putting it outside of the const messes it up.  This was the only way we were able to get it to work.